<!--%Tag(FULL)%-->
<launch>
  
	<param name="bextract_args" value=""/>
	<param name="experiment_data_dir" value="/home/haikalpribadi/Workspace/ROS/radiophonic/dir/experiment" />
	<param name="feature_set" value="31"/>
  <param name="feature_subs" value="1" />
  <param name="filter_peaks" value="false"/>
  <param name="input_mode" value="folder"/>
  <param name="gesture_topic" value="visualize_data" />
  <param name="gesture_range" value="70" />
  <param name="music_dir" value="/home/haikalpribadi/Workspace/ROS/radiophonic/dir/music" />
  <param name="normalizer_update_range" value="false" />
  <param name="osc_address" value="192.168.1.177" />
  <param name="osc_port" value="8888" />
  <param name="osc_offset" value="40" />
  <param name="output_to_file" value="false" />
  <param name="parameter_dir" value="/home/haikalpribadi/Workspace/ROS/radiophonic/dir/parameters" />
  <param name="pca_dimension" value="10" />
  <param name="pca_dir" value="/home/haikalpribadi/Workspace/ROS/radiophonic/dir/pca" />
  <param name="sample_duration" value="1" />
  <param name="sample_log" value="100" />
  <param name="transform_scale" value="1" />
  <param name="transform_mult" value="10" />
  <param name="visualizer_height" value="48" />
  <param name="visualizer_width" value="64" />
  <param name="visualizer_scale" value="2" />
  <param name="visualizer_data_rate" value="86" />
  <param name="wait_for_pca" value="true" />
  
  <param name="kinect_x_min" value="45"/>
	<param name="kinect_x_max" value="525"/>
  <param name="kinect_y_min" value="40"/>
  <param name="kinect_y_max" value="360"/>
  <param name="kinect_z_min" value="0"/>
  <param name="kinect_z_max" value="200"/>
  <param name="kinect_filter_size_n" value="5"/>
  <param name="kinect_smoothing" value="true"/>

  <node pkg="audiogesture" type="data_transformer" name="DataTransfomer" output="screen" />
  
  <node pkg="audiogesture" type="data_visualizer" name="DataVisualizer" output="screen" />
  
  <node pkg="audiogesture" type="gesture_visualizer" name="GestureVisualizer" output="screen"/>
  
  <node pkg="rviz" type="rviz" name="RViz" output="screen" />
  
</launch>
<!--%EndTag(FULL)%-->

<!--
  <node pkg="audiogesture" type="audio_input" name="AudioInput" output="screen"/>
  
-->
